index.d.ts

```diff
@@ -24,626 +24,6 @@ declare global {
-
-  interface LineProps extends BaseProps, Omit<WidgetJSX.LineProps, 'direction' | 'length'> {
-    length?: WidgetJSX.LineProps['length']
-  }
-
-  interface SVGProps extends BaseProps, Partial<WidgetJSX.FrameProps> {
-    src: string
-  }
-
-  interface BaseProps extends WidgetJSX.BaseProps {
-    // We have a custom onClick api that returns a promise
-    onClick?: (event: WidgetClickEvent) => Promise<any> | void
-    key?: string | number
-    tooltip?: string
-  }
-
-  interface HasChildrenProps {
-    children?: FigmaDeclarativeNode | FigmaDeclarativeNode[]
-  }
-  namespace WidgetJSX {
-    export interface Vector {
-      x: number
-      y: number
-    }
-
-    export interface Color {
-      r: number
-      g: number
-      b: number
-      a: number
-    }
-
-    export type AlignItems = 'center' | 'start' | 'end' | 'baseline'
-    export type BlendMode =
-      | 'normal'
-      | 'multiply'
-      | 'screen'
-      | 'overlay'
-      | 'darken'
-      | 'lighten'
-      | 'color-dodge'
-      | 'color-burn'
-      | 'hard-light'
-      | 'soft-light'
-      | 'difference'
-      | 'exclusion'
-      | 'hue'
-      | 'saturation'
-      | 'color'
-      | 'luminosity'
-
-    export type PaintType =
-      | 'image'
-      | 'solid'
-      | 'gradient-linear'
-      | 'gradient-radial'
-      | 'gradient-angular'
-      | 'gradient-diamond'
-
-    export interface PaintProps {
-      type: PaintType
-      blendMode?: BlendMode
-      visible?: boolean
-      opacity?: number
-    }
-
-    export interface SolidPaint extends PaintProps {
-      type: 'solid'
-      color: Color | HexCode
-    }
-
-    export interface ColorStop {
-      position: number
-      color: Color
-    }
-
-    export interface GradientPaint extends PaintProps {
-      type: 'gradient-linear' | 'gradient-radial' | 'gradient-angular' | 'gradient-diamond'
-      gradientHandlePositions: [Vector, Vector, Vector]
-      gradientStops: ColorStop[]
-    }
-
-    export type Transform = [[number, number, number], [number, number, number]]
-
-    export interface ImagePaint extends PaintProps {
-      type: 'image'
-      src: string
-      imageSize?: { width: number; height: number }
-      scaleMode?: ScaleMode
-      imageTransform?: Transform
-      scalingFactor?: number
-      rotation?: number
-      imageRef?: string
-    }
-
-    export type Paint = SolidPaint | GradientPaint | ImagePaint
-
-    export interface DropShadowEffect {
-      type: 'drop-shadow'
-      color: HexCode | Color
-      offset: Vector
-      blur: number
-      blendMode?: BlendMode
-      spread?: number
-      visible?: boolean
-      showShadowBehindNode?: boolean
-    }
-
-    export interface InnerShadowEffect {
-      type: 'inner-shadow'
-      color: HexCode | Color
-      offset: Vector
-      blur: number
-      blendMode?: BlendMode
-      spread?: number
-      visible?: boolean
-    }
-
-    export type ShadowEffect = DropShadowEffect | InnerShadowEffect
```

Comment:
I think this is fine if it is too hard to add back but something we want to be mindful of - we're remove this type alias so anyone who uses the `ShadowEffect` type alias directly will now need to update their code when they update typings.

Reply:
I can add it back.